jena-arq/src/main/java/org/apache/jena/sparql/core/DatasetGraphBase.java

```diff
@@ -84,6 +84,7 @@ public Graph getUnionGraph() {
     @Override
     public abstract void removeGraph(Node graphName) ;
 
+    @Deprecated
```

Comment:
Do we need to suggest an alternative in a javadoc deprecated? Or is it unlikely or not supposed to be used by other devs?